tail.go

```diff
@@ -126,6 +129,7 @@ func TailFile(filename string, config Config) (*Tail, error) {
 		Filename: filename,
 		Lines:    make(chan *Line),
 		Config:   config,
+		lineBuf:  new(strings.Builder),
```

Comment:
Instead of:
```
t := &Tail{
		Filename: filename,
		Lines:    make(chan *Line),
		Config:   config,
		lineBuf:  new(strings.Builder),
	}
```

```
t := &Tail{
		Filename: filename,
		Lines:    make(chan *Line),
		Config:   config,
	}

if config.CompleteLines {
    t.lineBuf = new(strings.Builder)
}
```
Then below only add the behaviour after checking t.lineBuf != nil